docs/containers/debug-node.md

```diff
@@ -82,21 +60,34 @@ Some Node.js application frameworks include CLIs for managing the application an
 }
 ```
 
+## Automatically launching the browser to the entry page of the application
+
+The Docker extension can automatically launch the browser to the entry point of the application after it has started in the debugger. This feature is enabled by default and configured via the `dockerServerReadyAction` object of the debug configuration in `launch.json`.
+
+This feature depends on several aspects of the application:
+
+ - The application must output logs to the debug console.
+ - The application must log a "server ready" message.
+ - The application must serve a browsable page.
+
+While the default settings may work for an Express.js based application, other Node.js frameworks may require explicit configuration of one or more of those aspects.
+
 ### Ensuring application logs are written to the debug console
 
-The Docker extension can automatically launch the browser to the entry point of the application after it has started in the debugger. (This feature is enabled by default and configured via the `dockerServerReadyAction` object of the debug configuration in `launch.json`.) This feature depends on the application writing its logs to the debug console of the attached debugger.  However, not all logging frameworks will write to the debug console, even when configured to use a console-based logger (as some "console" loggers will actually bypass the console and write directly to `stdout`).
+This feature depends on the application writing its logs to the debug console of the attached debugger.  However, not all logging frameworks will write to the debug console, even when configured to use a console-based logger (as some "console" loggers will actually bypass the console and write directly to `stdout`).
```

Comment:
more "will write", "will actually bypass" => rewrite in present tense

Reply:
Updated.